src/components/shared/loadingSpinner/LoadingSpinner.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './loadingSpinner.css';
+
+const LoadingSpinner = (props) => {
+    return <div className='loader' style={{ color: props.color }}></div>;
```

Comment:
actually please to something like style={{...props.styles}} 

we dont have to necessarry pass only color. Probably we will also need to set custom styling like height/widh (what ever we want)